core/controlplane/config/templates/cluster.yaml

```diff
@@ -239,7 +239,7 @@ worker:
 #      # If omitted, public subnets are created by kube-aws and used for worker nodes
 #      subnets:
 #      - # References subnets defined under the top-level `subnets` key by their names
```

Comment:
yes - when uncommenting the line, we need the " - ". Pls check another location like line 65 for comparison